.travis.yml

```diff
@@ -1,8 +1,9 @@
 language: d
+
 d:
-  - dmd
-  - ldc
-sudo: false
+  - dmd-2.090.1
+  - ldc-1.20.0
```

Comment:
This will get outdated very soon.

Reply:
We need a new Vibe.d release for that.

Reply:
Ah. I didn't realize this. Sorry!

Reply:
For reference, this is the required PR to get things to build with Vibe.d and > 2.090 FE again: https://github.com/vibe-d/vibe.d/pull/2439

Reply:
I guess we could have used the beta: https://github.com/vibe-d/vibe.d/releases/tag/v0.9.0-beta.1